wincode/src/schema/impls.rs

```diff
@@ -1081,11 +1089,11 @@ macro_rules! impl_seq {
     };
 }
 
-impl_seq! { "alloc", BTreeMap<K: Ord, V> }
-impl_seq! { "std", HashMap<K: Hash | Eq, V> }
-impl_seq! { "alloc", BTreeSet<K: Ord> }
-impl_seq! { "std", HashSet<K: Hash | Eq> }
-impl_seq! { "alloc", LinkedList<K:> }
+impl_seq! { "alloc", BTreeMap<K: Ord, V>, |_| BTreeMap::new() }
+impl_seq! { "std", HashMap<K: Hash | Eq, V>, |len| HashMap::with_capacity(len) }
```

Comment:
I guess it could be passed as a function instead of using lambda:
`impl_seq! { "std", HashMap<K: Hash | Eq, V>, HashMap::with_capacity }`

Reply:
and maybe this would fix the lints that you had to allow, does it?